packages/devtools_app/lib/src/screens/inspector_v2/inspector_tree_controller.dart

```diff
@@ -179,8 +183,17 @@ class InspectorTreeController extends DisposableController
 
   late final InspectorTreeConfig config;
 
-  set selection(InspectorTreeNode? node) {
-    if (node == _selection) return;
+  /// Refreshes the tree's rows if the return value of the [updateTreeAction]
+  /// callback is true.
+  void refreshTree(bool Function() updateTreeAction) {
```

Comment:
make updateTreeAction required and named so that callers of refreshTree read a little easier